frontend/benefit/azure-pipelines-master.yml

```diff
@@ -10,6 +10,8 @@ trigger:
     - frontend/*
     exclude:
     - README.md
+    - frontend/**/browser-tests
```

Comment:
no mneed to deploy new version on platta if something under `browser-tests` or `__tests__` has changed